lib/edools/lesson_progress.rb

```diff
@@ -0,0 +1,148 @@
+# frozen_string_literal: true
+
+module Edools
+  class LessonProgress
+    attr_reader :lesson_progress, :paginate, :errors
+
+    def initialize(data = {})
+      @lesson_progress = objectify_lesson_progress(data)
+      @errors = data.dig(:errors)
+    end
+
+    def self.find_paginated_from_enrollments(enrollment_id: , page: 1, per_page: 50)
```

Comment:
personally (I don't know if I can use this here lol), I don't like magic numbers.

I would use `DEFAULT_PAGE` and `DEFAULT_PER_PAGE`

Reply:
Makes sense and won't hurt, also next the constant definition we can add comments explaining why such numbers were used.